docs/contribution/profiling.md

```diff
@@ -53,7 +53,7 @@ $ go install github.com/google/pprof@latest
 
 ## Link gperftools Libraries When Compiling
 
-The library needed to run the CPU profilier should be linked into the ACTS project using the following build option:
+The library needed to run the CPU profiler should be linked into the Acts project using the following build option:

```

Comment:
I think we converged on *ACTS* over *Acts*.

Reply:
I thought the other way around.
Let's figure out and I will adapt all of them in this PR.

Reply:
I think we decided `Acts` for the namespace and `ACTS` everywhere else.